cmake/kokkos_enable_options.cmake

```diff
@@ -68,11 +70,20 @@ FUNCTION(check_device_specific_options)
         ENDIF()
       ENDIF()
       IF(KOKKOS_ENABLE_${OPTION})
-        MESSAGE(FATAL_ERROR "Kokkos_ENABLE_${OPTION} is ON but ${SOME_DEVICE} backend is not enabled.")
+        IF (KOKKOS_SKIP_VALIDATION)
+          MESSAGE(STATUS "${SOME_DEVICE} is not active - turning off ${OPTION}")
+          SET(KOKKOS_ENABLE_${OPTION} OFF PARENT_SCOPE) #turn it off, don't error out
```

Comment:
Even though I agree with the logic here.  This is causing the scripts to print a diff between `KokkosCore_config.h` from a generated makefile and cmake.

Reply:
So... change to a warning that this will have no effect?

Reply:
I take it back you are doing the right thing here.